package.json

```diff
@@ -171,8 +171,7 @@
     "common-tags": "^1.8.2",
     "deepmerge": "^4.2.2",
     "eslint": "^8.23.0",
-    "eslint-config-stylelint": "^15.1.0",
-    "eslint-plugin-jest": "^27.0.1",
+    "eslint-config-stylelint": "^16.0.0",
```

Comment:
[note] I've removed `eslint-plugin-jest` from the top-level dependencies (04b561a6) to prevent the installation of different package versions.

`eslint-config-stylelint` depends on and installs automatically `eslint-plugin-jest`:

```console
$ npm v eslint-config-stylelint dependencies
{
  'eslint-config-prettier': '^8.5.0',
  'eslint-plugin-jest': '^27.0.1',
  'eslint-plugin-node': '^11.1.0',
  'eslint-plugin-regexp': '^1.9.0'
}
```